tests/integration/security/sds_egress/sds_istio_mutual_egress_test.go

```diff
@@ -50,6 +50,7 @@ func TestSdsEgressGatewayIstioMutual(t *testing.T) {
 	// Turn it back on once issue is fixed.
 	t.Skip("https://github.com/istio/istio/issues/17933")
 	framework.NewTest(t).
+		Features("security.egress.mtls.sds").
```

Comment:
do we add this flag even for skipped test?

Reply:
I think we need to add it because this test is skipped because it if flaky. so when this test has been fixed. this feature needs to be enabled